README.md

```diff
@@ -37,6 +39,14 @@ Alternatively, you can download a [library archive](https://github.com/tlaplus/C
 Being a community-driven repository puts the community in charge to check the validity and correctness of submissions. The maintainers of this repository will try to keep this place in order, but we can't guarantee the quality of the
 modules and therefore cannot provide any assistance on eventual malfunctions.
 
+Test
```

Comment:
This should probably be moved below the Contributing section.

Reply:
Yes, I've thought that you may test before contributing (or that they don't have a specific order), buy I've added it below the Contributing section, thanks!